src/codeu/chat/client/commandline/Chat.java

```diff
@@ -34,6 +34,7 @@
 import codeu.chat.common.InterestSet;
 import codeu.chat.common.ServerInfo;
 import codeu.chat.common.User;
+import codeu.chat.server.Controller;
```

Comment:
You should delete this since 1) I don't see it used anywhere and 2) if you're working in a client file, never import a server class and vice versa, everything just gets really disordered and confusing